package/lab.yaml

```diff
@@ -0,0 +1,8 @@
+# Include this class only on test clusters
+classes:
```

Comment:
The idea here is, that appuio lab will use that configuration (and hence default to master branch) instead of the latest tag)

The version of the component however, can still be overridden in the tenant configuration (e.g to a branch to test a feature)

Reply:
As discussed we will remove the `lab.yaml` file and will just move the config to the `syn-tenant-repo`